lib/src/access.ts

```diff
@@ -29,59 +18,18 @@ export type RewrapResponse = {
  */
 export async function fetchWrappedKey(
   url: string,
-  requestBody: RewrapRequest,
-  authProvider: AuthProvider,
-  clientVersion: string
+  signedRequestToken: string,
+  authProvider: AuthProvider
 ): Promise<RewrapResponse> {
-  const req = await authProvider.withCreds({
-    url,
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(requestBody),
-  });
-
-  let response: Response;
-
+  const platformUrl = getHostFromEndpoint(url);
```

Comment:
To get the key for the unrwap request the `url` is coming from the tdf manifest. The url uses `/kas` as prefix `https://host.com/kas`.
The issue is that to create platform transport we require the url or the platform, not the kas.

So not really sure whats the right way to do this